Makefile

```diff
@@ -0,0 +1,17 @@
+CXXFLAGS += -I include/mapbox/vector_tile --std=c++14 -Wall -Wextra -Werror 
```

Comment:
@mapsam: it should be `-std=c++11` by default. So, only one hyphen and defaulting to c++11 instead of c++14. Can you fix this upstream too where you are pulling from?